apps/institution-service/src/models/widgetPreferences.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Association,
+  CreationOptional,
+  DataTypes,
+  ForeignKey,
+  Model,
+} from "sequelize";
+import sequelize from "../database";
+import { Aggregator } from "./aggregator";
+
+export class WidgetPreferences extends Model {
+  declare id: string;
+
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+
+  declare defaultAggregatorId: ForeignKey<Aggregator["id"]>;
+
+  declare static associations: {
+    defaultAggregator: Association<WidgetPreferences, Aggregator>;
+  };
+}
+
+WidgetPreferences.init(
+  {
+    id: {
+      type: DataTypes.STRING,
+      unique: true,
+      primaryKey: true,
```

Comment:
It might need to be updated here too
```suggestion
      type: DataTypes.UUID,
      unique: true,
      primaryKey: true,
      defaultValue: DataTypes.UUIDV4,
```